bin/run_swiftlint

```diff
@@ -14,7 +14,7 @@ while [[ "$#" -gt 0 ]]; do
   esac
 done
 
-if [[ -n $1 ]]; then
+if [[ -n ${1+x} ]]; then
```

Comment:
That `+x` feels really odd. At first I thought it was some special bash syntax but I'm not seeing it in https://devhints.io/bash and I think I confused it with `${1:-}` which is what I'd use to get an empty string instead of undefined variable error:

```suggestion
if [[ -n ${1:-} ]]; then
```
Or alternatively you can check for the number of arguments instead of checking if `$1` is empty/undefined:
```suggestion
if [[ $# -gt 0 ]]; then
```

Reply:
Ah, I like checking for the number of arguments (matching also the `while` on top) better! Nice suggestion, thanks!

Reply:
Btw worth double checking if it's supposed to be 0 or 1: I always forget if $0 (aka the program name) is counted in `$#` and didn't test my suggestion

Reply:
I tested to double check a bit before commiting: `$0` doesn't count in `$#`.